drivers/flash/spi_nor.c

```diff
@@ -338,14 +338,14 @@ static int spi_nor_configure(struct device *dev)
 	struct spi_nor_data *data = dev->driver_data;
 	const struct spi_nor_config *params = dev->config->config_info;
 
-	data->spi = device_get_binding(CONFIG_SPI_NOR_SPI_NAME);
+	data->spi = device_get_binding(DT_SPI_NOR_SPI_NAME);
 	if (!data->spi) {
 		return -EINVAL;
 	}
 
-	data->spi_cfg.frequency = CONFIG_SPI_NOR_SPI_FREQ_0;
+	data->spi_cfg.frequency = DT_SPI_NOR_SPI_FREQ_0;
 	data->spi_cfg.operation = SPI_WORD_SET(8);
-	data->spi_cfg.slave = CONFIG_SPI_NOR_SPI_SLAVE;
+	data->spi_cfg.slave = DT_SPI_NOR_SPI_SLAVE;
```

Comment:
Please change the name `slave` to something more appropriate.

Reply:
see https://github.com/zephyrproject-rtos/zephyr/pull/11850#discussion_r244771987.

Reply:
`struct spi_config` in `include/spi.h`, defines slave as
slave is the slave number from 0 to host controller slave limit.

In this PR, is the value being assigned a slave ID or a memory address?

Reply:
It is not a memory address but a slave ID (something that designates the total number of slaves from 0 to whatever number). You want it to be renamed to DT_SPI_NOR_SPI_SLAVE_ID?

Reply:
yes, `DT_SPI_NOR_SPI_SLAVE_ID` would be better. I got misled by the DTS generated name containing `SPI_FLASH_0_BASE_ADDRESS`

Reply:
Done.